pkg/statistics/handle/util/util.go

```diff
@@ -85,6 +86,8 @@ func CallWithSCtx(pool util.SessionPool, f func(sctx sessionctx.Context) error,
 	defer func() {
 		if err == nil { // only recycle when no error
 			pool.Put(se)
+		} else {
+			infosync.DeleteInternalSession(se)
```

Comment:
I think it is better to put the session in the pool after cleaning up the dirty state..

Reply:
but how to clean it(